src/ui/shared/CardHeader.tsx

```diff
@@ -25,19 +26,38 @@ import AuthorAndTimestampLabel from '../AuthorAndTimestampLabel';
 import resolutions from '../feedbackResolutions';
 import { CARD_HEADER_HEIGHT } from './../constants';
 import FeedbackContextType from 'fontoxml-feedback/src/FeedbackContextType';
+import { AnnotationErrorType } from 'fontoxml-review-reference-configuration/src/types';
 
-function determineShareButtonLabel(reviewAnnotation, error, isLoading) {
+function determineShareButtonLabel(
+	reviewAnnotation: CardContentComponentProps['reviewAnnotation'],
+	error: AnnotationErrorType,
+	isLoading: boolean)
+{
 	if (isLoading) {
 		return t('Sharing…');
 	}
 
 	return reviewAnnotation.busyState === BusyState.SHARING &&
-		error &&
-		error.recovery === RecoveryOption.RETRYABLE
+		typeof error !== 'number' &&
+		error?.recovery === RecoveryOption.RETRYABLE
 		? t('Retry share')
 		: t('Share');
 }
 
+type Props = {
+	context: CardContentComponentProps['context'];
+	hasReplyInNonIdleBusyState: boolean;
+	isSelectedToShare: CardContentComponentProps['isSelectedToShare'];
+	onReviewAnnotationEdit: CardContentComponentProps['onReviewAnnotationEdit'];
+	onReviewAnnotationRemove: CardContentComponentProps['onReviewAnnotationRemove'];
+	onReviewAnnotationResolve: CardContentComponentProps['onReviewAnnotationResolve'];
+	onReviewAnnotationShare: CardContentComponentProps['onReviewAnnotationShare'];
+	onReviewAnnotationShareAddRemoveToggle: CardContentComponentProps['onReviewAnnotationShareAddRemoveToggle'];
+	onReviewAnnotationShowInCreatedContext: CardContentComponentProps['onReviewAnnotationShowInCreatedContext'];
+	onReviewAnnotationShowInResolvedContext: CardContentComponentProps['onReviewAnnotationShowInResolvedContext'];
+	reviewAnnotation: CardContentComponentProps['reviewAnnotation'];
```

Comment:
Set this type to `CardContentComponentProps['reviewAnnotation'] & { targets: Target[] }`.

Reply:
You are right, and also targets should be optional.